test-framework/core/src/main/java/org/keycloak/test/framework/server/KeycloakServerConfigBuilder.java

```diff
@@ -1,82 +1,101 @@
 package org.keycloak.test.framework.server;
 
+import io.quarkus.maven.dependency.Dependency;
+import io.quarkus.maven.dependency.DependencyBuilder;
 import io.smallrye.config.SmallRyeConfig;
 import org.eclipse.microprofile.config.spi.ConfigSource;
+import org.keycloak.common.Profile;
 
+import java.util.Arrays;
 import java.util.HashMap;
 import java.util.HashSet;
 import java.util.LinkedList;
 import java.util.List;
 import java.util.Map;
 import java.util.Set;
+import java.util.stream.Collectors;
 
-public class CommandBuilder {
+public class KeycloakServerConfigBuilder {
 
     private final String command;
     private final Map<String, String> options = new HashMap<>();
     private final Set<String> features = new HashSet<>();
     private final Set<String> featuresDisabled = new HashSet<>();
     private final LogBuilder log = new LogBuilder();
+    private final Set<Dependency> dependencies = new HashSet<>();
 
-    private CommandBuilder(String command) {
+    private KeycloakServerConfigBuilder(String command) {
         this.command = command;
     }
 
-    public static CommandBuilder startDev() {
-        return new CommandBuilder("start-dev");
+    public static KeycloakServerConfigBuilder startDev() {
+        return new KeycloakServerConfigBuilder("start-dev");
     }
 
-    public CommandBuilder bootstrapAdminClient(String clientId, String clientSecret) {
+    public KeycloakServerConfigBuilder bootstrapAdminClient(String clientId, String clientSecret) {
         return option("bootstrap-admin-client-id", clientId)
                 .option("bootstrap-admin-client-secret", clientSecret);
     }
 
-    public CommandBuilder cache(String cache) {
+    public KeycloakServerConfigBuilder cache(String cache) {
         return option("cache", cache);
     }
 
     public LogBuilder log() {
         return log;
     }
 
-    public CommandBuilder features(Set<String> features) {
-        this.features.addAll(features);
+    public KeycloakServerConfigBuilder features(Profile.Feature... features) {
+        this.features.addAll(toFeatureStrings(features));
         return this;
     }
 
-    public CommandBuilder featuresDisabled(Set<String> featuresDisabled) {
-        this.featuresDisabled.addAll(featuresDisabled);
+    public KeycloakServerConfigBuilder featuresDisabled(Profile.Feature... features) {
```

Comment:
So, you propose rather using Java params then String values, am I right?

Reply:
Yes, a enum is less error prone as it allows only using valid values